apps/ethvm/src/components/blocks/largeBlocks/blockDetail.vue

```diff
@@ -216,11 +216,10 @@ export default Vue.extend({
   mounted() {
     this.setItems()
     this.setMore()
-    console.log(this.uncles)
   },
   computed: {
     isUncle() {
-      return this.block.getIsUncle()
+      return false
```

Comment:
Why is this removed?

Reply:
previously in getBlock was giving both Block and uncle, now there is a separate call to get uncle, so all block which are here are always Block not uncle